test/output/0300/no-ns-chart.config

```diff
@@ -21,3 +23,7 @@ fileConfig:
   test/output/0300/no-ns-chart-generated/my-role-role.yaml: {}
   test/output/0300/no-ns-chart-generated/my-rolebinding-rb.yaml: {}
   test/output/0300/no-ns-chart-generated/my-service-account-sa.yaml: {}
+  test/output/0300/no-ns-chart-generated/sample-0300-controller-manager-metrics-monitor-servicemonitor.yaml:
+    "":
```

Comment:
`XPathRoot` looks evident in the code, but using `""` here may confuse the users. Do you have any better ideas?

Reply:
I thought about this at the time. The issue we have is that this key in the config represents an XPath, and that the required XPath is root. 
Considered something muckier like `.` or `/` to represent root which would have to be resolved to `""` as the config is loaded, however these characters may still need to be quoted.
Could possibly use `root` as a key, since AFAIK that doesn't appear as a root level property on any known resource, but that's not to stop someone creating it as part of a CRD.